tools/latest-kernel-full-config.sh

```diff
@@ -2,17 +2,7 @@
 
```

Comment:
Let's update the `usage` to make it clear now that the functions provided by this script are intended to be run inside the bottlerocket-sdk